docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+
+  lisk-explorer:
+    image: lisk/explorer:${EXPLORER_TAG}
```

Comment:
Maybe `latest` is good enough for most end users? We could make a not in the README that it can be replaced with a specific version number if one wishes.